endpointmanager/pkg/capabilityhandler/capabilityreceiver.go

```diff
@@ -47,12 +51,21 @@ func formatMessage(message []byte) (*endpointmanager.FHIREndpoint, error) {
 		originalURL = url
 	}
 
+	capInt, ok := msgJSON["capabilityStatement"].(map[string]interface{})
+	if !ok {
+		return nil, fmt.Errorf("unable to cast capability statement to map[string]interface{}")
+	}
```

Comment:
I'd recommend getting rid of this check or changing it somehow. With this here, if an error was thrown while trying to get the capability statement, and therefore the cap stat is nil, then it exits out of formatMessage with no FHIREndpoint and therefore in saveMsgInDB nothing gets saved in the database. It seems like ideally you would still save all the metadata even if there is no capability statement. Also, NewCapabilityStatementFromInterface checks if capInt is nil anyways, so when I commented out the above line and ran it saved the data with no issue.